tests/bundles_test.go

```diff
@@ -101,7 +101,7 @@ var _ = Describe("kairos bundles test", Label("bundles"), func() {
 				out, _ := vm.Sudo("cat /oem/90_custom.yaml")
 				// https://pkg.go.dev/regexp/syntax
 				// ?s -> "let . match \n (default false)"
-				Expect(out).ToNot(MatchRegexp("(?s)ttl.sh.*ttl.sh"))
+				Expect(out).ToNot(MatchRegexp("(?s)quay.io/kairos/ci-temp-images.*quay.io/kairos/ci-temp-images"))
```

Comment:
I think we need an additional check that the entry is there once. This is easy to become false-positive (e.g. if we add `dimitris` to the regexp, it will always be green. It wasn't better before, so feel free to fix or postpone for other PR.

Reply:
done